src/backend/web/static/swagger/api_v3.json

```diff
@@ -8160,6 +8163,401 @@
+                      "type": "boolean"
+                    },
+                    "NodeH": {
+                      "type": "boolean"
+                    },
+                    "NodeI": {
+                      "type": "boolean"
+                    },
+                    "NodeJ": {
+                      "type": "boolean"
+                    },
+                    "NodeK": {
+                      "type": "boolean"
+                    },
+                    "NodeL": {
+                      "type": "boolean"
+                    }
+                  }
+                },
+                "BotRow": {
+                  "type": "object",
+                  "required": [
+                    "NodeA", "NodeB", "NodeC", "NodeD", "NodeE", "NodeF", "NodeG", "NodeH", "NodeI", "NodeJ", "NodeK", "NodeL"],
+                  "properties": {
+                    "NodeA": {
+                      "type": "boolean"
+                    },
+                    "NodeB": {
+                      "type": "boolean"
+                    },
+                    "NodeC": {
+                      "type": "boolean"
+                    },
+                    "NodeD": {
+                      "type": "boolean"
+                    },
+                    "NodeE": {
+                      "type": "boolean"
+                    },
+                    "NodeF": {
+                      "type": "boolean"
+                    },
+                    "NodeG": {
+                      "type": "boolean"
+                    },
+                    "NodeH": {
+                      "type": "boolean"
+                    },
+                    "NodeI": {
+                      "type": "boolean"
+                    },
+                    "NodeJ": {
+                      "type": "boolean"
+                    },
+                    "NodeK": {
+                      "type": "boolean"
+                    },
+                    "NodeL": {
+                      "type": "boolean"
+                    }
+                  }
+                },
+                "Trough": {
+                  "type": "integer"
+                }
+              }
+          },
+          "algaePoints": {
+            "type": "integer"
+          },
+          "netAlgaeCount": {
+            "type": "integer"
+          },
+          "wallAlgaeCount": {
+            "type": "integer"
+          },
+          "endGameBargePoints": {
+            "type": "integer"
+          },
+          "autoBonusAchieved": {
+            "type": "boolean"
+          },
+          "coralBonusAchieved": {
+            "type": "boolean"
+          },
+          "bargeBonusAchieved": {
+            "type": "boolean"
+          },
+          "coopertitionCriteriaMet": {
+            "type": "boolean"
+          },
+          "coopertitionBonusAchieved": {
+            "type": "boolean"
+          },
+          "foulCount": {
+            "type": "integer"
+          },
+          "techFoulCount": {
```

Comment:
Double check that these are still "tech fouls", considering the manual now uses the language "major foul"